components/WotwRandoSettings.vue

```diff
@@ -325,6 +329,33 @@
       document.removeEventListener('keydown', this.onKeyDown)
     },
     methods: {
+      enableDevTools(){
+        this.settings['Flags.Dev'] = true
+        this.debugStreak = 0
+
+        setTimeout(() => {
+          this.$refs.developerSettings.scrollIntoView({
+          behavior: 'smooth',
+          block: 'nearest',
+          })
+        })
+        
+      },
+      disableDevTools(){
+        this.settings['Flags.Dev'] = false
+      },
+      hostChanged(){
+        setTimeout(() => {
+          switch(this.settings["Paths.Host"]){
+            case "wotw.orirando.com":
+              this.settings["Paths.UdpPort"] = "31415"
+              break
+            case "dev.wotw.orirando.com":
+              this.settings["Paths.UdpPort"] = "31416"
+              break
+          }
+        })
+      },
```

Comment:
I would only want to keep this temporarily for the following reasons:

- The moment you switch between beta/non-beta it already switches both Host and UdpPort for you
- We don't want to do stuff like "play non-beta on beta server" again because it sucks :D
- In the long term, I want to get rid of the UdpPort setting completely. The client has to connect to the WebSocket before anyways to get the encryption key for the UDP communication, and at that point the server can just tell the client which UdpPort it should use for that session